Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptEditController.java

```diff
@@ -216,13 +206,79 @@ public String confirmlEdit(@PathVariable("id") String id,
 				}
 			}
 		}
-
 		ConceptEntry[] arraySynonyms = new ConceptEntry[synonyms.size()];
+
 		int i = 0;
+		StringBuffer jsonStringBuilder = new StringBuffer("{");

```

Comment:
Ideally you want all of the JSON code out of this method. Any reason you have this here and not in it's own method?